crates/astria-conductor/src/executor.rs

```diff
@@ -219,11 +252,64 @@ impl<C: ExecutionClient> Executor<C> {
         let timestamp = convert_tendermint_to_prost_timestamp(block.header.time)
             .wrap_err("failed parsing str as protobuf timestamp")?;
 
+        // send the data in the sequencer block to the execution client
         let response = self
             .execution_rpc_client
-            .call_do_block(prev_block_hash, txs, Some(timestamp))
+            .call_do_block(prev_execution_block_hash, txs, Some(timestamp))
             .await?;
         self.execution_state = response.block_hash.clone();
+        // TODO pending (GHI-205): reevaluate this after 205 gets merged
+        self.last_executed_seq_block_hash = Hash::try_from(block.block_hash.clone()).unwrap();
```

Comment:
why unwrap?

Reply:
The `Hash::try_from` returns an Result. I guess I could an `if let` here instead.